system_baseline/models.py

```diff
@@ -158,8 +159,13 @@ def delete_by_system_ids(cls, system_ids, account_number, org_id):
         db.session.commit()
 
     @classmethod
-    def get_mapped_system_count(cls, account_number, org_id):
-        query = cls.query.with_entities(
+    def get_mapped_system_count(cls, account_number, org_id, rbac_group_filters=None):
+        mapped_systems_query = cls.query
+        if rbac_group_filters:
```

Comment:
I'd recommend to do a stricter check for `is None`, to handle case of an empty array.